### DISCUSSION TITLE
Préstamo maldito

### DISCUSSION THREAD

**Discussion by zebnat** [zebnat/morpho-dimension#20] [Ideas]
Propuesta: Préstamo maldito 

El usuario podrá pedir un préstamo, pero su cuenta quedará maldita. El préstamo deberá ser devuelto con un interés que dependerá de la cantidad de ese préstamo. En caso de no devolver el dinero en ese plazo, tu cuenta puede perder capacidades.

## Ejemplo

El usuario `soypobreayuda` decide pedir un préstamo de 500,000 macromonedas a devolver en 30 días, por tanto la comisión será del 20%. Si el usuario no devuelve 500,000 + 100,000 macromonedas en menos de 30 días, su cuenta sería penalizada.

## Condiciones

- Requisitos de nivel para utilizar el sistema.
- Dependiendo de tu nivel podrás pedir diferentes tipos de préstamo.
- No se pueden pedir nuevos préstamos sin pagar el anterior.
- No se puede transferir dinero hasta que tu préstamo esté pagado.

## Penalizaciones

- Perder 1/4 1/3 o 1/2 del nivel actual.
- Perder capacidades críticas de uso de la página como podría ser `comentar en perfiles`, `comentar en juegos`.
- Tus avatares podrían ser reseteados.
- ...

## ¿En qué se basará el tipo de penalización?

Dependiendo del % del préstamo no devuelto, la penalización a tu cuenta será mucho mayor. Por ejemplo, pides `500,000` y te conceden 60 días para devolver. El día que finaliza el tiempo permitido has devuelto un total de `450,000` así pues restarían `50,000`.

Se penalizaría tu cuenta en función de esos `50,000` no devueltos. El préstamo es maldito así que no sería recomendable arriesgarse, ya que como he dicho, te puede penalizar duramente o resetearte avatares. Aún así, puede ser un reto el hecho de tener que pagarlo a tiempo.

**Comment by Darkusito** [comment]
USUARIO SOY POBRE AYUDA AJKSZJKAJKAKJASJKJKAJK ¿Qué intentas decirnos, Zeb? >:(

xDD Fuera de eso, es bastante buena idea. Ponlo ya qué ocupo esos avatares de Fnaf. uwu

**Comment by max466666** [comment]


**Reply by zebnat** [reply]
Creo que esto no procede aquí

**Comment by Chelsita** [comment]
Muy buena idea Zeb, ojala se implemente pronto, ando en pobreza absoluta jajaja xd

**Reply by zebnat** [reply]
Ojo, bienvenida a github. ¿Vas a ser programadora? La idea es un poco loca, pero todo en orden viniendo de mi.

**Comment by AngelHiramPR** [comment]
Lo unico q veo absurdo es lo de no poder comentar en perfiles ni en juegos

**Reply by zebnat** [reply]
Pues tienes razón. Es un tanto absurdo, aún así sería un riesgo que podrías correr 😈.

**Comment by Lince31** [comment]
El riesgo es mínimo, me gusta la idea. :)

**Comment by Sebas9001** [comment]
Mi Nombre En Macrojuegos Es Sebastian9001 Y Estoy De Acuerdo Con El préstamo Maldito 💎💖☄️

**Comment by TheMiler** [comment]
Buenas. Una opinión que me gustaría brindar es que, al pedir un préstamo y que no se haya pagado, una de las penalizaciones sería quitarle el 100% del dinero prestado o un X porcentaje . Por ejemplo X persona pide un préstamo de 500.000 macromonedas, al pasar los días acordado en el préstamo, se le descontarían 400.000 macromonedas (o menos). La persona quedaría en números negativos y tendrá que recuperarlos. 
 Esta penalización serviría para evitar que se hagan prestamos, se gasten las monedas y nunca perderían monedas. También serviría para evitar que suceda lo siguiente: X usuario es de nivel 100, Pide un préstamo, se gasta el dinero y no pierde monedas. Además le dan otra penalización, que es que le bajen a nivel 50. Esto les beneficia ya que podría volver a subir a nivel 100 y ganar muchas monedas. Ya que ahora es fácil subir de nivel, muchos harían esto.

 Lo siento por hacerlo tan largo, espero que se haya entendido mi opinión y que puedan tener ideas con mi comentario. 
Un saludo! Miler1000

**Comment by Sebas9001** [comment]
Me gusta la Opinion menos mal soy lvl 94 :v

**Comment by SoySebastian9001** [comment]
Hola soy sebastian9001 me cree una nueva cuenta aqui en GitHub Porq se me olvido la contra de la otra cuenta

**Comment by SoySebastian9001** [comment]
Quiero Hacer Un Prestamo Soy El Usuario Sebastian9001 Zebnat Prestame 700.000 Y te las devuelvo en 30 diaas o en menos de 300 dias @zebnat

**Comment by SoySebastian9001** [comment]
Quiero Hacer Un Prestamo Admin @zebnat  Podria Prestarme 700.000
